dressup/src/pages/ProductsPage/FilterFunctions.js

```diff
@@ -0,0 +1,112 @@
+import { ProductsPageData } from "./ProductsPage.data";
+
+const sortProductList = (order, products) => {
+  switch (order) {
+    case "highToLow":
+      return products.sort(
+        (a, b) => b.price.discountPrice - a.price.discountPrice
+      );
+    case "lowToHigh":
+      return products.sort(
+        (a, b) => a.price.discountPrice - b.price.discountPrice
+      );
+    default:
+      return products;
```

Comment:
.sort() mutates the original array. Either make a copy of the array or use spread operator!